modules/op-restarting-and-canceling-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,66 @@
+// This module is included in the following assemblies:
+// * pac/managing-pipeline-runs-pac.adoc
+
+:_content-type: PROCEDURE
+[id="restarting-and-canceling-pipeline-run-using-pipelines-as-code_{context}"]
+= Restarting and canceling a pipeline run using {pac}
+
+You can restart or cancel a pipeline run with no events, such as sending a new commit to your branch or raising a pull request. To restart all pipeline runs, use the *Re-run all checks* feature in a GitHub App. To restart all or specific pipeline runs, use the following comments:
+
+* The `/test` and `/retest` comment restarts all pipeline runs.
+
+* The `/test <pipeline_run_name>` and `/retest <pipeline_run_name>` comment restarts a specific pipeline run.
+
+To cancel all or specific pipeline runs, use the following comments:
+
+* The `/cancel` comment cancels all pipeline runs.
+
+* The `/cancel <pipeline_run_name>` comment cancels a specific pipeline run.
+
+The results of the comments are visible under the *Checks* tab of a GitHub App.
+
+.Procedure
+
+* If you use a GitHub App, go to the *Checks* tab and click *Re-run all checks*.
```

Comment:
`*Re-run all checks*` is only available for Pull Request 
so should we mention that ?

Reply:
@savitaashture sounds good, I'll quickly make that change.